### PR TITLE
Update gseapy to 0.12.1

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`gseapy`](https://bioconda.github.io/recipes/gseapy/README.html): **0.12.0** &rarr; **0.12.1**

[install with bioconda](http://bioconda.github.io/recipes/gseapy/README.html) [Conda](https://anaconda.org/bioconda/gseapy/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/gseapy`](https://github.com//bioconda/bioconda-recipes/tree/bump/gseapy/recipes/gseapy) (click to view/edit other files)
Summary | Gene Set Enrichment Analysis in Python
Home | [https://github.com/zqfang/gseapy](https://github.com/zqfang/gseapy)
Releases |[https://pypi.org/pypi/gseapy/json](https://pypi.org/pypi/gseapy/json)

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

File: recipes/gseapy/meta.yaml [hunk 1]
```diff
@@ -1,45 +1,44 @@
...
+  build: 
+    - {{ compiler('rust') }}
   host:
-    - rust
-    - python >3
+    - python
+    - wheel
     - setuptools
     - setuptools-rust
     - numpy >=1.13.0
```
Comment (marcelm) [inline recipes/gseapy/meta.yaml:23]: The `host:` section should usually only list dependencies required to run `pip install`, so you can most likely omit `scipy`, `pandas` etc. which are only needed at runtime.

Comment (marcelm) [reply recipes/gseapy/meta.yaml]: There were some strange CI errors, so I took the liberty of pushing these changes also to re-trigger CI.

Comment (zqfang) [reply recipes/gseapy/meta.yaml]: Any thougths to make MacOS build work ? I managered to build it on my MacOS with bioconda-utils.

Comment (marcelm) [reply recipes/gseapy/meta.yaml]: Sorry, I have no idea. The log output is unfortunately very unhelpful. Let’s focus on #36469 instead.

**Pull request closed without merging**